notebooks/asian_barrier_option/elu_activation/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.8 FATAL_ERROR)
+project(cmake_and_cuda LANGUAGES CXX CUDA)
+
+set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS}  \
+--expt-relaxed-constexpr \
+--expt-extended-lambda \
+-gencode arch=compute_70,code=sm_70 \
+-gencode arch=compute_75,code=sm_75 \
+-Wno-deprecated-declarations")
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-deprecated-declarations")
+
+set(BERT_LIBS
```

Comment:
I would change BERT naming since this is not BERT. Call it `ELU_LIBS`.

Reply:
done